code/database/migrations/2022_08_13_223049_update_users_model.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('lastname')->after('name');
+            $table->string('username')->after('lastname');
+            $table->renameColumn('name', 'firstname');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
```

Comment:
Quita el docblock, y agrega el tipo de retorno indicadolo con declaracion de tipo ( https://www.php.net/manual/es/language.types.declarations.php )

```suggestion
    public function down(): void
```

Con esto el bloque de @return void es redundante y la validación estará implicita en el código permitiendo que PHP simule un tipado estatico, lo mismo con el metodo down